CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Supercontrast

```

Comment:
It looks like your changes here overlap with your changes in your other PR, so I am going to leave all my comments on your other PR :)